SofaKernel/modules/SofaLoader/src/SofaLoader/MeshOBJLoader.h

```diff
@@ -30,15 +30,15 @@ namespace sofa::component::loader
 
 using sofa::core::objectmodel::BaseData;
 
-class SOFA_SOFALOADER_API MeshObjLoader : public sofa::core::loader::MeshLoader
+class SOFA_SOFALOADER_API MeshOBJLoader : public sofa::core::loader::MeshLoader
```

Comment:
TODO: add a class alias MeshObjLoader -> MeshOBJLoader + use ATTRIBUTE_DEPRECATED macro on it

Reply:
Deprecation period: v21.12 -> v22.12